edenia/antelope-tools
# Chore/improve UI homepage 1095

### Improve the homepage for wide screens

### What does this PR do?

- Resolve #1095, resolves #1064, 
  resolves #1065
- Replace the image.
- When the width grows more than 1920px the content is centered.

### Steps to test

1. Open the docs/index.html.
2. Check the responsive design.

### Screenshots

#### **Width over 1024px**
![imagen](https://user-images.githubusercontent.com/66583677/207966902-7e279931-7f49-446b-8483-2e4573a4e84b.png)

#### **Width over 1920px**
![imagen](https://user-images.githubusercontent.com/66583677/207966781-f57a5d3f-9a66-4531-be59-302dfc8adb6a.png)
![imagen](https://user-images.githubusercontent.com/66583677/207966652-fbb5cfac-33ba-42f5-8752-028f246b97d7.png)
